src/api.ts

```diff
@@ -399,6 +416,9 @@ class RippleAPI extends EventEmitter {
   computeLedgerHash = computeLedgerHash // @deprecated Invoke from top-level package instead
   signPaymentChannelClaim = signPaymentChannelClaim // @deprecated Invoke from top-level package instead
   verifyPaymentChannelClaim = verifyPaymentChannelClaim // @deprecated Invoke from top-level package instead
+
+  generateFaucetWallet = generateFaucetWallet
```

Comment:
It needs the rippled connection to make sure the wallet has been funded properly.

Reply:
Hi @mvadari I'm not sure to get what you mean here. Is there something missing in the code?

Reply:
This was in response to someone else's comment which isn't visible anymore for some reason.